networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/subnets/AttestationTopicSubscriber.java

```diff
@@ -96,16 +96,16 @@ public synchronized void subscribeToPersistentSubnets(
     boolean shouldUpdateENR = false;
 
     for (SubnetSubscription subnetSubscription : newSubscriptions) {
-      int subnetId = subnetSubscription.getSubnetId();
+      int subnetId = subnetSubscription.subnetId();
```

Comment:
When accessing the field you can directly use `subnetSubscription.subnetId`. The function is useful when a lambda is required.

Reply:
ah fair - only my second record i've converted :D

Reply:
oh this one got changed by the refactor! thats why i didnt recognise it

Reply:
i intellij won't allow it to be selected, so i'm going to leave it... its only listing fns...